metals/src/main/scala/scala/meta/internal/metals/GithubNewIssue.scala

```diff
@@ -0,0 +1,50 @@
+package scala.meta.internal.metals
+
+import java.net.URLEncoder
+
+import scala.util.Properties
+
+import org.eclipse.lsp4j.ClientInfo
+
+object GithubNewIssue {
+  def buildUrl(clientInfo: ClientInfo): String = {
+    val clientVersion =
+      if (clientInfo.getVersion() == null) ""
+      else s" ${clientInfo.getVersion()}"
+    val body =
+      s"""|<!--
+          |        Describe the bug ...
+          |
+          |        Reproduction steps
+          |          1. Go to ...
+          |          2. Click on ...
+          |          3. Scroll down to ...
+          |          4. See error
+          |-->
+          |
+          |### Expected behaviour:
+          |
+          |<!-- A clear and concise description of what you expected to happen. -->
+          |
+          |**Operating system:**
+          |${Properties.osName}
+          |
+          |**Java version:**
+          |${Properties.javaVersion}
+          |
+          |**Editor/extension:**
+          |${clientInfo.getName()}$clientVersion
+          |
```

Comment:
We could also add info about the build tool used in the workspace and the vuild server currently used? You can find out how it's used in the Metals doctor. And maybe Scala versions used in the workspace?

We can have a section "Workspace Info" or something like that